build.gradle.kts

```diff
@@ -0,0 +1,64 @@
+plugins {
+    id("java-library")
+    id("com.github.johnrengelman.shadow") version("8.1.1")
+}
+
+java {
+    toolchain.languageVersion.set(JavaLanguageVersion.of(21))
+}
+
+group = "de.stealwonders"
+version = "1.2"
+
+val paperVersion = "1.20.4-R0.1-SNAPSHOT"
+val hikariVersion = "5.1.0"
+val mariadbClientVersion = "3.3.2"
+
+repositories {
+    mavenCentral()
+    maven { url = uri("https://papermc.io/repo/repository/maven-public/") } // Paper
+    maven { url = uri( "https://repo.aikar.co/content/groups/aikar/") }
+    maven { url = uri("https://repo.destroystokyo.com/repository/maven-public/") }
+    maven { url = uri("https://oss.sonatype.org/content/groups/public/") }
+    maven { url = uri("https://www.iani.de/nexus/content/groups/public") }
+}
+
+dependencies {
+    compileOnly("io.papermc.paper:paper-api:$paperVersion")
+    compileOnly("de.iani.cubeside:PlayerUUIDCache:1.5.1-SNAPSHOT")
```

Comment:
Shouldn't depend on this anymore, `Bukkit.getOfflinePlayerIfCached` should suffice.

Reply:
Done